src/core/schema/nodes.js

```diff
@@ -152,6 +152,28 @@ export default {
 	},
 
 
+	math_display: {
+		group: 'block math',
+		content: 'text*',
+		atom: true,
+		code: true,
+		toDOM: node => ['pre', {
+			class: 'math'
+		}, '$$' + node.textContent + '$$'],
+		parseDOM: [{
+			tag: 'pre.math',
+			getContent(dom) {
```

Comment:
I forgot to write it the last time, but it should be `getContent(dom, schema) {` as described [here](https://prosemirror.net/docs/ref/#model.ParseRule.getContent).

We should also add `import { Fragment } from 'prosemirror-model';` at the top.